test/smoke/rh-docker/perm_test.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook:: nexus_iq_server
+#
+# Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
+#
+
+# Permissions test for recipe nexus_iq_server::rh-docker
+
+control 'rh-docker-perms-test-001' do
+  title 'Allows nexus user to mutate /etc/password'
```

Comment:
technically `/etc/passwd` or it could be written as "...mutate system password file" or something like that.